src/include/miopen/solver_id.hpp

```diff
@@ -51,6 +51,7 @@ enum class Primitive
     Batchnorm,
     Bias,
     Fusion,
+    Item,
```

Comment:
I don't know if the numeric values for these show up anywhere (such as in log files or database), but unless you have a good reason it's generally better to only add items to the end of the enumeration.
One exception would be alphabetical order, but it's clear this isn't being followed here.

Reply:
fixed it.